lib/utils/utils.ts

```diff
@@ -115,6 +115,18 @@ export const derivePairId = (pair: Pair) => {
   return `${pair.baseCurrency}/${pair.quoteCurrency}`;
 };
 
+/**
+ * Convert ticker symbole into pair.
```

Comment:
If we keep this function, this comment has a typo and should say something like "Derives the base and quote currencies from a trading pair ticker."

Reply:
We will remove it.